dev/devicelab/bin/tasks/system_debug_ios.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/ios.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+
+import 'service_extensions_test.dart';
+
+Future<void> main() async {
+  deviceOperatingSystem = DeviceOperatingSystem.ios;
+  await task(() async {
+    final Device device = await devices.workingDevice;
+    await device.unlock();
+    final String deviceId = device.deviceId;
+
+    final Directory testDirectory =
+        dir('${flutterDirectory.path}/dev/integration_tests/ui');
+    await inDirectory<void>(testDirectory, () async {
+      await flutter('packages', options: <String>['get']);
+      await prepareProvisioningCertificates(testDirectory.path);
+
+      await checkNoWarningHostLaunch(deviceId);
+      await checkNoWarningXcodeLaunch(deviceId);
+      await checkWarningHomeScreenLaunch(deviceId);
+    });
+    return TaskResult.success(<String, dynamic>{});
+  });
+}
+
+// When a debug-mode app is launched from the host there should be no warnings.
+Future<void> checkNoWarningHostLaunch(String deviceId) async {
+  final String output = await evalFlutter('drive', options: <String>[
+    '--debug',
+    '--verbose',
+    '--verbose-system-logs',
+    '--local-engine=ios_debug',
```

Comment:
Does this require a local engine build?

Reply:
No, that was for local testing, sorry!
However, the tests won't pass until the corresponding CL has been landed in the Engine.